blueocean-dashboard/src/main/less/run-message-cell.less

```diff
@@ -1,12 +1,45 @@
 
 .RunMessageCell {
     display: flex;
+    width: 100%;
+    height: 100%;
     align-items: center;
 
-    .RunMessageCellInner {
+    .RunMessageCellInner,
+    .RunMessageCellInner > span {
+        display: flex;
         flex: 1;
-        white-space: nowrap;
-        overflow: hidden;
-        text-overflow: ellipsis;
+        width: 100%;
+        height: 100%;
+
+        a {
+            display: flex;
+            flex: 0 0 auto;
+            align-items: center;
+            height: 100%;
+            min-width: 0;
+
+            &:last-child {
+                flex-grow: 1;
+                white-space: nowrap;
+                text-overflow: ellipsis;
+                overflow: hidden;
+
+                &:not(:first-child) > span{
+                    width: 10px;
+                }
+
+                > span {
```

Comment:
Seems like rather than applying this rule to the `span` we should use a semantic className here so the rule reads better, and so it's clearer in the JSX why we are wrapping the text values in the first place. Something like `.ellipsis-text` would probably be fine?